### PR TITLE
fix: find-open-source-project broken link�🔗

### LINKED ISSUE #3020: [BUG] the README.md in the support section has a link to "finding-open-source-projects.md" which is showing "Page not found" error
### Description of the bug

*link: https://github.com/EddieHubCommunity/support/blob/main/tips/finding-open-source-projects.md 
is it supposed to be updated? If yes, then how can I update it?*

### Working Environment (e.g. operating system, browser, device)

browser

### Please add screenshots (if applicable)

buggg

bugg

__This is my first time raising an issue. Thanks for all the support @EddieHubCommunity__

### PR DESCRIPTION
**Pull request by pranv-sonawane**
closes #3020

#### What does this PR do?
This fixed the broken link in the README.md file present in the support section
#### Description of the task to be completed?
Yes

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Learn more on [GitHub docs](https://docs.github.com/en/github/setting-up-and-man
   Here is a list of resources that also might help you start contributing to open source:
   - [How to get started in hackathons](https://github.com/EddieHubCommunity/support/issues/32)
   - [Open source tips from EddieBot](https://github.com/EddieHubCommunity/support/tree/main/tips)
```
Comment (Panquesito7) [inline README.md:73]: This is wrong.
```suggestion
  - [Open source tips from EddieBot](https://github.com/EddieHubCommunity/support/tree/main/docs/tips)
```

**Review by Pradumnasaraf — APPROVED** [review]
LGTM :)

**Review by Panquesito7 — APPROVED** [review]

**Review by krishguptadev — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -70,8 +70,8 @@ Learn more on [GitHub docs](https://docs.github.com/en/github/setting-up-and-man
 
   Here is a list of resources that also might help you start contributing to open source:
   - [How to get started in hackathons](https://github.com/EddieHubCommunity/support/issues/32)
-  - [Open source tips from EddieBot](https://github.com/EddieHubCommunity/support/tree/main/tips)
-- **How to look for an open-source project to contribute?** Check [Finding Open Source Projects](./tips/finding-open-source-projects.md)
+  - [Open source tips from EddieBot](https://github.com/EddieHubCommunity/support/tree/main/docs/tips)
```
Comment (schmelto) [inline README.md:73]:
```suggestion
  - [Open source tips from EddieBot](./docs/tips)
```

File: README.md [hunk 3]
```diff
@@ -70,8 +70,8 @@ Learn more on [GitHub docs](https://docs.github.com/en/github/setting-up-and-man
...
+- **How to look for an open-source project to contribute?** Check [Finding Open Source Projects](https://github.com/EddieHubCommunity/support/blob/main/docs/tips/finding-open-source-projects.md)
```
Comment (schmelto) [inline README.md:74]:
```suggestion
- **How to look for an open-source project to contribute?** Check [Finding Open Source Projects](./docs/tips/finding-open-source-projects.md)
```

**Comment by schmelto** [comment]
I would consider making the links relativ

**Review by Pradumnasaraf — COMMENTED** [review]
@schmelto Correct, Adding Repo relative is much best-case scenario.

**Review by krishguptadev — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by nhcarrigan — APPROVED** [review]
Looks good to me.

**Pull request merged**